### ISSUE TITLE
Working with older versions of raven-js

### ISSUE THREAD

**Issue by grimurd** [gdi2290/angular-raven#18]
We haven't upgraded sentry to version 8 yet but when updating one of our angular projects i noticed that this library had updated to the latest raven-js release.

I'm still using version 0.5.11 intentionally because i wanted to keep using the older release in order to use sentry 7 but it still updated to the latest version. 

In the bower.json for this project, raven-js is set to "*" so it always installs the latest release.

This is quite inconvenient for us using older releases.

I suggest doing a new 0.5 release and setting the raven-js entry to  "~1.3.0"

**Comment by jpatel3** [comment]
+1

**Comment by PatrickJS** [comment]
would it be better if I just removed the dependencies in package.json

**Comment by jpatel3** [comment]
@gdi2290 other way we resolved it by explicitly adding raven-js version in our package.json

```
    "angular-raven": "0.6.2",
    "raven-js": "1.3.0",
```

**Comment by PatrickJS** [comment]
yeah, that's the correct way to doing that. Can you make a PR to the readme to explain this?
